confuse.py

```diff
@@ -134,36 +135,193 @@ def __init__(self, filename, reason=None):
 
 # Views and sources.
 
+
+UNSET = object()  # sentinel
+
+
+def _load_first(func):
+    '''Call self.load() before the function is called - used for lazy source
```

Comment:
Can we stick with the [PEP 8-recommended](https://www.python.org/dev/peps/pep-0008/#documentation-strings) triple-double quotes for docstrings?

Reply:
Yeah sure. I don't like the style of unnecessary double quotes when single will do, but since it's pep8, ok.